specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/virtualMachine.json

```diff
@@ -852,6 +855,12 @@
           },
           "Get a virtual machine with VM Size Properties": {
             "$ref": "./examples/virtualMachineExamples/VirtualMachine_Get_WithVMSizeProperties.json"
+          },
+          "Get a virtual machine with DiskController Type Properties": {
+            "$ref": "./examples/virtualMachineExamples/VirtualMachine_Get_WithDiskControllerType.json"
+          },
+          "Get a virtual machine with instance view and DiskController Type Properties": {
+            "$ref": "./examples/virtualMachineExamples/VirtualMachine_Get_VM_With_InstanceView_WithDiskControllerType.json"
```

Comment:
Let use keep the example file name and description generic as it captures all the mainline VM properties and instance view, so rename the description to Get a virtual machine with instance view and also rename the file name to VirtualMachine_Get_VM_WithInstanceView.json